### PR TITLE
Feature/storybook-theme

### PR DESCRIPTION
**Pull request by dlnr**

### CODE REVIEW

File: storybook/storybook-theme/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (alimpens) [inline storybook/storybook-theme/package.json:1]: I'm not sure we should make a seperate package for this. If we add `@storybook/theming` to both storybooks and just use this directory to store a generic theme config, we could keep it a lot more simple. Or maybe even easier: I think we only need to define the theme in the main storybook (storybook-css), right? Then we could do everything in that directory.

File: storybook/storybook-theme/amsterdam.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { create } from '@storybook/theming/create'
+import logo from './logo.svg'
```
Comment (alimpens) [inline storybook/storybook-theme/amsterdam.js:2]: Can't we get the logo from `@amsterdam/icons`?

Comment (alimpens) [reply storybook/storybook-theme/amsterdam.js:2]: Sure

File: storybook/storybook-theme/package.json (hunk 1 above)
Comment (dlnr) [reply storybook/storybook-theme/package.json:1]: I had to add npm to be able to use theming, the alternative is the storybook directories having their own theme config.

Comment (dlnr) [reply storybook/storybook-theme/package.json:1]: I can remove main though

Comment (alimpens) [reply storybook/storybook-theme/package.json:1]: Maybe we can just store the config in `/storybook` (the object with `brandTitle` and `brandImage`), and create the themes seperately in both storybooks.

File: proprietary/icons/svgo.config.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+module.exports = {
+  plugins: [
+    {
+      name: 'preset-default',
+      params: {
+        overrides: {
+          inlineStyles: false,
+          convertStyleToAttrs: false,
+          removeStyleElement: false,
+          removeDoctype: false,
```
[the remark this answers is not in the document]
Comment (dlnr) [reply proprietary/icons/svgo.config.js:11]: Removed

File: storybook/storybook-theme/package.json (hunk 1 above)
Comment (dlnr) [reply storybook/storybook-theme/package.json]: Changed it to suggested solution

Comment (alimpens) [reply storybook/storybook-theme/package.json]: One suggestion (last one, I swear ;) ) : you suggest a `/storybook-theme` directory with one file in it. Couldn't we just place a file directly in the `/storybook` directory, and call it `themeConfig.js` or something?

**Review by alimpens — APPROVED** [review]

**Review by alimpens — APPROVED** [review]

**Pull request merged**